src/LndClient/Data/PendingChannel.hs

```diff
@@ -30,13 +30,20 @@ instance
         (GRPC.pendingChannelsResponse_PendingChannelRemoteNodePub x)
       <*> channelPointParser
         (GRPC.pendingChannelsResponse_PendingChannelChannelPoint x)
-      <*> fromGrpc
-        (GRPC.pendingChannelsResponse_PendingChannelCapacity x)
-      <*> fromGrpc
-        (GRPC.pendingChannelsResponse_PendingChannelLocalBalance x)
-      <*> fromGrpc
-        (GRPC.pendingChannelsResponse_PendingChannelRemoteBalance x)
-      <*> fromGrpc
-        (GRPC.pendingChannelsResponse_PendingChannelLocalChanReserveSat x)
-      <*> fromGrpc
-        (GRPC.pendingChannelsResponse_PendingChannelRemoteChanReserveSat x)
+      <*> (toMSat <$> fromGrpc (GRPC.pendingChannelsResponse_PendingChannelCapacity x))
+      <*> ( toMSat
+              <$> fromGrpc
+                (GRPC.pendingChannelsResponse_PendingChannelLocalBalance x)
+          )
+      <*> ( toMSat
+              <$> fromGrpc
+                (GRPC.pendingChannelsResponse_PendingChannelRemoteBalance x)
+          )
+      <*> ( toMSat
+              <$> fromGrpc
+                (GRPC.pendingChannelsResponse_PendingChannelLocalChanReserveSat x)
+          )
+      <*> ( toMSat
+              <$> fromGrpc
+                (GRPC.pendingChannelsResponse_PendingChannelRemoteChanReserveSat x)
```

Comment:
are there mSat equivalents for these fields?

Reply:
no, for this field there only satoshi field
https://api.lightning.community/#lnrpc-pendingchannelsresponse-pendingchannel

Reply:
ok!